backend/src/main/java/com/rota/api/StaffService.java

```diff
@@ -95,6 +95,32 @@
         .collect(Collectors.toList());
   }
 
```

Comment:
Should add some Javadoc comments here

Reply:
It's a private function, so it won't show in Javadoc anyway (which is why checkstyle didn't enforce it).
I can add a comment if you think it will help understand the code.

Reply:
Oh yeah of course! Didn't notice that. Completely up to you.